src/test/kotlin/com/terraformation/backend/customer/model/PermissionTest.kt

```diff
@@ -1074,6 +1074,7 @@ internal class PermissionTest : DatabaseTest() {
         setTestClock = true,
         updateAppVersions = true,
         updateDeviceTemplates = true,
+        updateGlobalRoles = true,
```

Comment:
add this for device manager user test as well?

Reply:
The pattern in this test class is to omit the `false` values since that's the default. The parameter list becomes, effectively, a list of the permissions that should be present.